README.md

```diff
@@ -1,4 +1,3 @@
-[//]: # (TODO: technical writers)
 # Codacy IntelliJ IDEA Extension
 
 The Codacy extension for IntelliJ IDEA helps you review and manage the issues found by Codacy directly within the IntelliJ IDE. It notifies you whether a pull request is up to standards by highlighting problematic code patterns and displaying code quality metrics.
```

Comment:
Is this for IntelliJ IDEA only? I tried installing it on other platforms (specifically [Writerside](https://www.jetbrains.com/writerside/)) and it seems to work OK. Any limitations on this?

Reply:
it "should" work for all intellij platform IDEs; eventually we can take a look into it, but I think we also tried it on webstorm and it worked (but installing the zip package)